src/resample.c

```diff
@@ -35,7 +35,8 @@ main (int argc, char *argv [])
 	int normalize = 1 ;
 	sf_count_t	count ;
 	double		src_ratio = -1.0, gain = 1.0 ;
-	int			new_sample_rate = -1, k, converter, max_speed = SF_FALSE ;
+	int			new_sample_rate = -1, k, converter, max_speed = SF_FALSE, raw_bits ;
```

Comment:
Now is a good time to fix the indentation on that line.

Indentation should be with tabs where a single tab is the same as four spaces.

Reply:
fixed